Update Ansible Steering Committee doc (#46)
ansible/community-docs: ansible_community_steering_committee.rst

Ansible Community Steering Committee
************************************

Mission:
========

The Steering Committee’s mission (hereinafter the Committee) is to provide continuity, guidance, and
suggestions to the Ansible community, to ensure delivery and high quality of the Ansible package.

The Committee helps decide the technical direction of the Ansible project and is responsible for approval of new

> proposals and policies in the community, package, and collections' world, new collection-inclusion requests,

Reviewer: ```suggestion
proposals and policies in the community, package, and collections world, new collection-inclusion requests,
```
very much a tiny nit, but don't think this should be possessive

> The Committee helps decide the technical direction of the Ansible project and is also responsible for avowal of new
> proposals and policies in the community, package, and collections' world, new collection-inclusion requests,
> and other technical aspects regarding inclusion and packaging.

Reviewer 2: `avowal` might not be the best term for an international audience, how about something like

```suggestion
The Committee helps decide the technical direction of the Ansible project. Its primary focus is to ensure that:
* proposals
* policies
* new collection inclusion requests
* (add other items here)
are adequately reviewed and progressed.
```

Author: I agree about `avowal` 100% - it confused me a bit when i saw it the first time:)
I've replaced it with `approval`. Is it OK?
I think the bullet point list would be an extra thing here because we have it a bit below describing things in detail. I would keep the `mission` paragraph as general as possible.

The Committee should reflect the scope and breadth of the Ansible Community.

The Committee Responsibilities:
===============================

The committee:

> * Designs policies and procedures for the collection world; vote on approval changes to established policies and procedures.

Reviewer: I'm not sure about this but should be we consider saying the "collection community" instead of the "collection world" throughout the document? Or do folks feel that is overusing or overloading the term "community"?

Reviewer 3: I vote for using community since the steering committee doesn't establish policies etc for certified collections. Worth making the distinction here.

Author: Good catch! I've changed `collections` to `community collections` everywhere.

* Reviews collections for compliance with the policies. 
* Helps create and define roadmaps for our deliverables such as the ``ansible`` package, major collections, and documentation.

> * Reviews collections submitted for inclusion in the Ansible package and decides whether include them or not.

Reviewer: ```suggestion
* Reviews collections submitted for inclusion in the Ansible package and decides whether to include them or not.
```

10. Dylan Silva (``thaumos``)
11. Brian Scholer (``briantist``)

Committee members are selected based on their active contribution to the Ansible Project and its Community.

See the `Steering Committee Membership Guidelines <https://github.com/ansible/community-docs/blob/main/steering_committee_membership_guidelines.rst>`_ to learn details.

New Policy/Proposals & Inclusion Requests:
==========================================

* The Committee uses the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ to asynchronously discuss with the Community and vote on Community topics in corresponding issues.

> * If you want to discuss an idea that can impact the community, collection best practices, inclusion policy, submitted new policy/proposals & new collection inclusion requests or other topics the Committee is responsible for, create a new issue in the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ as a discussion topic, and it’ll be discussed asynchronously in the issue itself.

Reviewer: I am struggling to come up with a proposed edit but I wonder if the list of things in this sentence is long enough to consider making it a bulleted list instead? Or maybe re-wording a little to make the list read easier.

Reviewer 3: Maybe something like :
Create a new issue in the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ as a discussion topic if you want to discuss an idea that impacts any of the following:
 - Ansible community
 - collection best practices
 - blablabla

Author: Agreed, thanks! Changed PTAL

* New collection inclusion requests are submitted through a new discussion in the `ansible-inclusion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_ repository.

Depending on a topic you want to discuss with the Community and the Committee, as you prepare your proposal, please consider the requirements established by:

* `Ansible Community Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#code-of-conduct>`_.
* `Ansible Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
* `Ansible Collection Inclusion Checklist <https://github.com/ansible-collections/overview/blob/main/collection_checklist.md>`_.

Meetings:
=========

See the Community Working Group meeting `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays>`_.

> Meeting summaries are posted in the `Community Working Group Meeting Agenda <https://github.com/ansible/community/issues/539>`_ issue.

Reviewer 4: Issue #539 tracked 2021 agendas.
Issue [#645](https://github.com/ansible/community/issues/645) tracks the agenda for meetings in 2022.

Author: Good catch! I've updated it with a link that uses labels, so it will always point to the current issue (i.e. there will be no need to update it every year). Thanks!
